config/pipeline.yaml

```
@@ -342,9 +361,91 @@ platforms:
 
   shell:
 
+  kontron-kswitch-d10-mmt-6g-2gs:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-6g-2gs.dtb'
+    boot_method: uboot
```

```suggestion
    <<: *armel-device
    mach: microchip
    dtb: dtbs/lan966x-kontron-kswitch-d10-mmt-6g-2gs.dtb
```

`armel-device` defines the necessary things for those devices using u-boot. We haven't yet formalized what `mach` should define in the new system, but using the vendor (as in, device-tree subfolder name) makes sense to me. Other opinions are welcome.